README.md

```diff
@@ -796,6 +796,8 @@ AWS Repos:
 
 Community Repos:
 
+* [Lumoslabs/aleph](https://github.com/lumoslabs/aleph) - Collaborative tool for query devepoment. Full featured web application for writing, running, explorign, and revision tracking across an entire organziation datbases.  Currently is focused on Redshift but has generic JDBC suppor and could work anwyere.  Only a month months old in open source but Has traction outside at other notable companies.
```

Comment:
I'm not sure about the new description change that went in?  I feel it is quite long and has multiple typos.

The requested changes were to fix the name and sorting, could we focus on those?  Thanks!